tests/unit/plugins/modules/test_proxmox_cluster.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2025, Florian Paul Azim Hoberg (@gyptazy) <florian.hoberg@credativ.de>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import sys
+import pytest
+
+proxmoxer = pytest.importorskip("proxmoxer")
+mandatory_py_version = pytest.mark.skipif(
+    sys.version_info < (2, 7),
+    reason="The proxmoxer dependency requires python2.7 or higher",
```

Comment:
Shouldn't this be 3.7, as discussed earlier in the issues?

Reply:
I'd completely remove this.

Reply:
I created #17 to clean up similar issues in other unit tests.

Reply:
will remove this as suggested by @felixfontein